project-base/src/Shopsys/ShopBundle/Resources/config/routing_front.yml

```diff
@@ -83,7 +83,7 @@ front_additional_image_without_type:
 
 front_login_check:
     path: /login/check/
-    # controller's action is unnecessary, because firewall processes whole request
+# controller's action is unnecessary, because firewall processes whole request
```

Comment:
This comment in all routing_front files should be probably before that route.

Reply:
fixed in 926ae8668